met-web/src/routes/UnauthenticatedRoutes.tsx

```diff
@@ -12,7 +12,7 @@ import { RedirectLogin } from './RedirectLogin';
 import withLanguageParam from './LanguageParam';
 import { Navigate, Route } from 'react-router-dom';
 import NotFound from './NotFound';
-import ViewEngagement, { engagementLoader } from 'components/engagement/new/view';
+import ViewEngagement, { engagementLoader } from 'components/engagement/new/view'; // TODO: Convert tenant calls to route loader
 import { SurveyLoader } from 'components/survey/building/SurveyLoader';
```

Comment:
Does this mean that tenant fetching should be deferred to the loader? Tenant info is already stored in the authentication slice

Reply:
This looks to be an existing comment, not something from me. It's possible that you removed it from the main branch when you uploaded your tenant loader commit, but it was still present on my instance, and thus you're seeing it here.